source/server/server_configuration_parser.cpp

```diff
@@ -17,6 +17,7 @@ namespace nidevice_grpc {
 static const char* kDefaultFilename = "server_config.json";
 static const char* kAddressJsonKey = "address";
 static const char* kPortJsonKey = "port";
+static const char* kSidebandIpJsonKey = "sideband_ip";
```

Comment:
I agree with the sideband_address name. 

regarding the port configuration, the sideband repo currently hardcodes it to [50055](https://github.com/ni/grpc-sideband/blob/b1cd688921128044054592315327b7768e7f894c/src/sideband_data.cc#L340C9-L340C50).

Would it be good idea to have another setting in the config file that has to be set to true to enable sideband (moniker service )? If enable_sideband is true in the config only then we sideband_address will make sense

Reply:
`port` can be added later, especially since this will be behind a toggle. I think it is an important feature, but is not needed "right now" at the beginning of dev.